src/test/java/org/broadinstitute/consent/http/db/DarCollectionDAOTest.java

```diff
@@ -243,13 +245,15 @@ public void testInsertDarCollectionNegative() {
         try {
             darCollectionDAO.insertDarCollection("darCode", 0, new Date());
         } catch (Exception e) {
-            assertEquals(PSQLState.FOREIGN_KEY_VIOLATION.getState(), ((PSQLException) e.getCause()).getSQLState());
+            assertEquals(PSQLState.FOREIGN_KEY_VIOLATION.getState(),
+                ((PSQLException) e.getCause()).getSQLState());
         }
         try {
             darCollectionDAO.insertDarCollection("darCode", userId, new Date());
             darCollectionDAO.insertDarCollection("darCode", userId, new Date());
         } catch (Exception e) {
-            assertEquals(PSQLState.UNIQUE_VIOLATION.getState(), ((PSQLException) e.getCause()).getSQLState());
+            assertEquals(PSQLState.UNIQUE_VIOLATION.getState(),
+                ((PSQLException) e.getCause()).getSQLState());
         }
     }
 
```

Comment:
nit: should these (and others in this file) be in an assertThrows?

Reply:
If this is the intent of the test, then this is more than a nit. As written, if the code under test doesn't throw an exception, the test will erroneously pass, since nothing is asserting that an exception must be thrown.

Reply:
I'm actively revisiting some of these as a separate task. There are more than a few cases of incorrect assertions. There is no technical code change in this PR, it's just a formatting change.